eng/helix/content/runtests.ps1

```diff
@@ -16,7 +16,7 @@ $currentDirectory = Get-Location
 $env:PLAYWRIGHT_BROWSERS_PATH = "$currentDirectory\ms-playwright"
 $env:PLAYWRIGHT_DRIVER_PATH = "$currentDirectory\.playwright\win-x64\native\playwright.cmd"
 
-$envPath = "$env:PATH;$env:HELIX_CORRELATION_PAYLOAD\node\bin"
+$env:Path = "$env:PATH;$env:HELIX_WORKITEM_ROOT;$env:HELIX_WORKITEM_ROOT\node\bin"
```

Comment:
This was way wrong before and we likely don't really need `node` in `$env:Path` on Windows